pkg/capr/planner/idempotent.go

```diff
@@ -36,7 +36,7 @@ fi
 `
 
 func idempotentActionScriptPath(controlPlane *rkev1.RKEControlPlane) string {
-	return path.Join(capr.GetProvisioningDataDir(controlPlane), "idempotence/idempotent.sh")
+	return path.Join(capr.GetProvisioningDataDir(&controlPlane.Spec.RKEClusterSpecCommon), "idempotence/idempotent.sh")
```

Comment:
I guess we don't need to worry if `controlPlane` is nil here, right?

Reply:
For all intents and purposes it can never be nil and reach this far.